From 69073ef05e6e0d0b5790075e98e38676790eca2c Mon Sep 17 00:00:00 2001
From: Tuzma-7 <tuz.ma@cyweemotion.com>
Date: Fri, 28 Jun 2024 10:33:32 +0800
Subject: [PATCH] patch_test

---
 cwm_test.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cwm_test.c b/cwm_test.c
index ecb2b1f..f0037d6 100755
--- a/cwm_test.c
+++ b/cwm_test.c
@@ -74,6 +74,7 @@ void test_print_calibration_value(void)
 void algo_test(void)
 {
     test_print_calibration_value();
+	CWM_OS_dbgPrintf("patch test");
 }
 
 
-- 
2.34.1.windows.1

